lib/plugins/globalid.js

```diff
@@ -6,10 +6,10 @@
 var jsonp = require('jsonp')
 var invariant = require('../utils/misc').invariant
 var noop = require('../utils/misc').noop
-var cookie = require('cookies-js')
+var cookie = require('js-cookies').default
 
 function cacheSuccess (result, cookieName) {
-  cookie.set(cookieName, result['global_id'], { expires: 6000 })
+  cookie.setItem(cookieName, result['global_id'], 6000)
```

Comment:
I don't think it's necessary -- this cookie isn't a source of truth for `td_global_id` like `td_client_id` is.